src/waveform/waveformwidgetfactory.cpp

```diff
@@ -604,15 +606,67 @@ bool WaveformWidgetFactory::widgetTypeSupportsUntilMark() const {
     return false;
 }
 
+bool WaveformWidgetFactory::widgetTypeSupportsStems() const {
+    switch (m_configType) {
+    case WaveformWidgetType::RGB:
+        return true;
+    case WaveformWidgetType::Filtered:
+        return true;
+    case WaveformWidgetType::Simple:
+        return true;
+    case WaveformWidgetType::HSV:
+        return true;
+    case WaveformWidgetType::Stacked:
+        return true;
```

Comment:
NIT: perfectly valid and idiomatic and less noisy
```suggestion
    case WaveformWidgetType::RGB:
    case WaveformWidgetType::Filtered:
    case WaveformWidgetType::Simple:
    case WaveformWidgetType::HSV:
    case WaveformWidgetType::Stacked:
        return true;
```

Reply:
Yeah, noticed that, too, but didn't touch because I assumed it's structured like this to allow adding exceptions in case renderer changes require so.
But of course that's easy to do and I'll make it more compact.